revisited_file.php

```diff
@@ -0,0 +1,92 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Tracking new or modified file for web service tool version. 
+ *
+ * @directory   classes/external
+ * @file        enqueue_merge_request.php (new)
+ * @description 
+ * 
+ * @directory   classes/external
+ * @file        get_data_merge_request.php (new)
+ * @description 
+ * 
+ * @directory   classes/
+ * @file        merge_request.php (new)
+ * @description 
+ * 
+ * @directory   classes/task
+ * @file        merge_user_accounts.php (new)
+ * @description 
+ * 
+ * @directory   classes/task
+ * @file        queue_to_process_pending_merge_request.php (new)
+ * @description 
+ * 
+ * @directory   db
+ * @file        install.xml (update)
+ * @description 
+ * 
+ * @directory   db
+ * @file        upgrade.php (update)
+ * @description 
+ * 
+ * @directory   db
+ * @file        services.php (new)
+ * @description 
+ * 
+ * @directory   db
+ * @file        tasks.php (new)
+ * @description 
+ * 
+ * @directory   lang/en
+ * @file        tool_mergeusers.php (update)
+ * @description 
+ * 
+ * @directory   
+ * @file        version.php (update)
+ * @description 
+ * 
+ * @directory   
+ * @file        CHANGES.md (update)
+ * @description 
+ * 
+ * @directory   
+ * @file        settings.php (update)
+ * @description
+ *  
+ * @directory   lib
+ * @file        mergeusertool.php (update)
+ * @description 
+ * 
+ * git status before the commit
+ *         modified:   CHANGES.md
+ *         new file:   classes/external/enqueue_merge_request.php
+ *         new file:   classes/external/get_data_merge_requests.php
+ *         new file:   classes/merge_request.php
+ *         new file:   classes/task/merge_user_accounts.php
+ *         new file:   classes/task/queue_to_process_pending_merge_requests.php
+ *         modified:   db/install.xml
+ *         new file:   db/services.php
+ *         new file:   db/tasks.php
+ *         modified:   db/upgrade.php
+ *         modified:   lang/en/tool_mergeusers.php
+ *         new file:   revisited_file.php
+ *         modified:   settings.php
+ *         modified:   version.php
+ *         modified:   lib/mergeusertool.php
+ */
```

Comment:
this file must not exist. revisit the patch before submitting or updating a PR, please.

Reply:
remove this file please.

Reply:
if you need it, you can have a local copy of it in another place, but remove it from the repository.